os/net/blemgr/bledev.c

```diff
@@ -124,6 +124,34 @@ int bledev_handle(struct bledev *dev, lwnl_req cmd, void *data, uint32_t data_le
 		TRBLE_DRV_CALL(ret, dev, del_bond_all, (dev, mode));
 	}
 	break;
+	case LWNL_REQ_BLE_CONN_IS_ACTIVE:
+	{
+		bool *is_active = NULL;
+		trble_conn_handle con_handle = 0;
+
+		lwnl_msg_params param = { 0, };
+		if (data != NULL) {
+			memcpy(&param, data, data_len);
```

Comment:
For using `memcpy`, please include `string.h`.

Reply:
Ok, i will add it